### PR TITLE
Add async verification validation rules

### PR DESCRIPTION
**Pull request by aghassemi**
Needed before we can remove the experiment. Follows the same rules as `action-xhr`

### CODE REVIEW

File: validator/validator-main.protoascii [hunk 1]
```diff
@@ -2558,6 +2558,19 @@ tags {  # <form method=GET ...>
     }
     blacklisted_value_regex: "__amp_source_origin"
   }
+  attrs: {
+    name: "verify-xhr"
```
Comment (honeybadgerdontcare) [inline validator/validator-main.protoascii]: nit: alphabetize the attributes.

File: validator/validator-main.protoascii [hunk 2]
```diff
@@ -2601,6 +2614,20 @@ tags {  # <form method=POST ...>
...
+    mandatory: true
```
Comment (honeybadgerdontcare) [inline validator/validator-main.protoascii]: Test is failing because of mandatory. Is this a mandatory_oneof instead?

Comment (aghassemi) [inline validator/validator-main.protoascii]: it's totally not mandatory :) thanks tests!

**Comment by Gregable** [comment]
This should be live in validation in about an hour.

**Pull request merged**